client/src/types/state.ts

```diff
@@ -0,0 +1,24 @@
+export type Build = {
+  id: string;
+  configurationId: string;
+  buildNumber: number;
+  commitMessage: string;
+  commitHash: string;
+  branchName: string;
+  authorName: string;
+  status: string;
```

Comment:
тут нужно написать тип на основе использования
```suggestion
  status: 'done' | 'close' | 'Waiting';
```

В целом - всегда нужно стараться сужать типы. То есть вместо `string`, где это возможно, нужно указывать объединение _определённых_ строк (или Enum)